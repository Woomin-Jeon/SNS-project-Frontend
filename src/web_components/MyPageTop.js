import React from 'react';

function MyPageTop({
  currentUserState,
}) {

  return (
    <div className="mypage-cover">
      <div className="mypage-cover-top">
        <img className="mypage-cover-profile-image" src={currentUserState.profile} alt="" />
        <span className="mypage-cover-username">{currentUserState.userName}</span>
      </div>
      <div className="mypage-cover-bottom">
        <button className="mypage-cover-word-timeline" type="button">타임라인</button>
        <button className="mypage-cover-word-information" type="button">정보</button>
        <button className="mypage-cover-word-friends" type="button">친구</button>
        <button className="mypage-cover-word-scrap" type="button">스크랩</button>
      </div>
    </div>
  );
}

export default MyPageTop;
