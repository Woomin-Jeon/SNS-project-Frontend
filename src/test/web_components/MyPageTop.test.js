import MyPageTop from "../../web_components/MyPageTop";
import React from "react";
import {shallow} from 'enzyme';

describe('MyPageTop', () => {
  let currentUserState;

  beforeEach(() => {
    currentUserState = {
      birth: '1996-04-21',
      location: '대한민국',
      email: 'dal96k@hanmail.net',
    }
  });

  it('renders well', () => {
    const component = shallow(
      <MyPageTop
        currentUserState={currentUserState}
      />
    );

    expect(component).toMatchSnapshot();
  });
});
