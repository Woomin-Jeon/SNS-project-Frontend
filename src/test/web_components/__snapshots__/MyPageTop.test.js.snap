// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyPageTop renders well 1`] = `ShallowWrapper {}`;
